### PR TITLE
Updating steps for k03 - RHSSO performance test case

### PR DESCRIPTION
**Pull request by davidffrench**
# Description
Updating steps of k03 - RHSSO performance test case for three reasons:
- An additional option for getting ec2 instance access
- Use the same keycloak version on cluster to run the performance tests
- Added workaround for running against RHSSO 7.4 (will work with 7.3 as well). This workaround will need to exist until 7.5 which likely won't be out for a while.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test-cases/tests/performance/k03-run-performance-test-against-rhsso.md [hunk 1]
```diff
@@ -11,31 +11,37 @@ Note: this test should only be performed at a time it will not affect other ongo
...
 
 ## Steps
-
-1. Login into [QE-AWS account](https://068334777414.signin.aws.amazon.com/console) using your AWS IAM account credentials and select the region to **London**.
-2. Create a new **t2.small** ec2 instance from the sso-load-testing AMI. Set the Auto-assign public IP to Enable and select Network as **sso-load-test-vpc** and subnet as **sso-load-test-subnet**. Configure security group to SSH with "myIP".
-3. SSH into the instance using the public IPv4 address and the centos user. `ssh -i <pemfilename>.pem centos@<public IP>`
-4. `cd keycloak` and `git pull`
+1. Get access to ec2 instance to run tests
+   -  Option 1 - Use existing ec2 instance:
+      1. Contact Stephin Thomas for details on accessing this ec2 instance
```
Comment (b1zzu) [inline test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:16]: @STEPHINRACHEL can we put the credentials for the ec2 instance in the [vault](https://gitlab.cee.redhat.com/integreatly-qe/vault), then we can just put the link to the vault here and we would not need a second option, wdyt?

**Comment by STEPHINRACHEL** [comment]
@b1zzu Sounds good.
https://gitlab.cee.redhat.com/integreatly-qe/vault#add-secrets-to-the-vault
Is this the step to add new secret?

**Comment by b1zzu** [comment]
> @b1zzu Sounds good.
> https://gitlab.cee.redhat.com/integreatly-qe/vault#add-secrets-to-the-vault
> Is this the step to add new secret?

Yes, you can also add them to the [SECRETS.md](https://gitlab.cee.redhat.com/integreatly-qe/vault/blob/master/SECRETS.md) file, is up to you :)

**Comment by pawelpaszki** [comment]
@b1zzu - can we merge this?

File: test-cases/tests/performance/k03-run-performance-test-against-rhsso.md [hunk 2]
```diff
@@ -11,31 +11,37 @@ Note: this test should only be performed at a time it will not affect other ongo
...
+   -  Option 2 - Create new ec2 instance:
+      1. Login into [QE-AWS account](https://068334777414.signin.aws.amazon.com/console) using your AWS IAM account credentials and select the region to **London**.
+      2. Create a new **t2.small** ec2 instance from the sso-load-testing AMI. Set the Auto-assign public IP to Enable and select Network as **sso-load-test-vpc** and subnet as **sso-load-test-subnet**. Configure security group to SSH with "myIP".
+2. SSH into the instance using the public IPv4 address and the centos user. `ssh -i <pemfilename>.pem centos@<public IP>`
+3. `cd keycloak` and `git fetch --tags`
+4. Checkout the Keycloak tag corespinding with RHSSO version on cluster
```
Comment (pawelpaszki) [inline test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:22]:
```suggestion
4. Checkout the Keycloak tag corresponding with RHSSO version on cluster
```

File: test-cases/tests/performance/k03-run-performance-test-against-rhsso.md [hunk 3]
```diff
@@ -11,31 +11,37 @@ Note: this test should only be performed at a time it will not affect other ongo
...
+9.  Run
    ```
    export WARMUP_PERIOD=900
    export RAMPUP_PERIOD=120
    export MEASUREMENT_PERIOD=240
    export STRESS_TEST_UPS_FIRST=14
    ./stress-test.sh -DmaxMeanReponseTime=1000
    ```
-10. Once tests have finished, results will be saved to `/home/centos/keycloak/testsuite/performance/tests/target/gatling/`. Copy the last two folders to your local machine to view the tests using something similar to `scp -r -i ../stephin.pem centos@ec2-3-8-28-1.eu-west-2.compute.amazonaws.com:/home/centos/keycloak/testsuite/performance/tests/target/gatling/oidcloginandlogoutsimulation-1580403162107 ./`
+12. Once tests have finished, results will be saved to `/home/centos/keycloak/testsuite/performance/tests/target/gatling/`. Copy the last two folders to your local machine to view the tests using something similar to `scp -r -i ../stephin.pem centos@ec2-3-8-28-1.eu-west-2.compute.amazonaws.com:/home/centos/keycloak/testsuite/performance/tests/target/gatling/oidcloginandlogoutsimulation-1580403162107 ./`
```
Comment (pawelpaszki) [inline test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:44]:
```suggestion
10. Once tests have finished, results will be saved to `/home/centos/keycloak/testsuite/performance/tests/target/gatling/`. Copy the last two folders to your local machine to view the tests using something similar to `scp -r -i ../stephin.pem centos@ec2-3-8-28-1.eu-west-2.compute.amazonaws.com:/home/centos/keycloak/testsuite/performance/tests/target/gatling/oidcloginandlogoutsimulation-1580403162107 ./`
```

**Comment by b1zzu** [comment]
@davidffrench can you run `make test-cases/fix` to fix the linter

**Comment by davidffrench** [comment]
> @davidffrench can you run `make test-cases/fix` to fix the linter

Done now @b1zzu

**Pull request merged**